src/webgpu/api/validation/render_pass/render_pass_descriptor.spec.ts

```diff
@@ -893,6 +908,7 @@ g.test('depth_stencil_attachment,depth_clear_value')
     u
       .combine('depthLoadOp', ['load', 'clear', undefined] as const)
       .combineWithParams([
+        { depthClearValue: undefined },
```

Comment:
I think this could just be simplified to:
```
.combine('depthClearValue', [undefined, -1.0, 0.0, 0.5, 1.0, 1.5])
```

Reply:
Thanks. I was wondering why it wasn't that way in the first place :P   Done